src/evm/tycho_db.rs

```diff
@@ -156,19 +147,14 @@ impl PreCachedDB {
     ///
     /// Returns an `Option` containing a reference to the storage value if it exists, otherwise
     /// returns `None`.
-    async fn get_storage_async(&self, address: &Address, index: &rU256) -> Option<rU256> {
+    pub fn get_storage(&self, address: &Address, index: &rU256) -> Option<rU256> {
         self.inner
             .read()
-            .await
+            .unwrap()
```

Comment:
can you explain why it is ok to use the `.unwrap()` here?? asking just to understand rust a bit better